packages/ui/resolution/ResolutionDecorator.js

```diff
@@ -97,7 +97,8 @@ const ResolutionDecorator = hoc(defaultConfig, (config, Wrapped) => {
 		static displayName = 'ResolutionDecorator';
 
 		static propTypes = /** @lends ui/resolution.ResolutionDecorator.prototype */ {
-			className: PropTypes.string
+			className: PropTypes.string,
+			fontScale: PropTypes.number
```

Comment:
Please add detailed docs and also set `fontScale` default value to be 1

Reply:
Added.